modules/cco-short-term-creds-aws-olm.adoc

```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// * authentication/managing_cloud_provider_credentials/cco-short-term-creds.adoc
+
+:_content-type: CONCEPT
+[id="cco-short-term-creds-aws-olm_{context}"]
+= OLM-managed Operator support for authentication with AWS STS
```

Comment:
It looks like this is first use of OLM in the assembly, so I would spell out the acronym on first use. Unless OLM is typically an exception to the rule in OpenShift docs as some acronyms can be, in which case, ignore me.

Reply:
Good point! I'll try to rephrase so we don't end up with "Operator Lifecycle Manager-managed Operator" :dizzy_face: